packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.jsx

```diff
@@ -1,55 +1,126 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Button, ThemedScrollbars, Carousel } from '@deriv/components';
+import { Button, ThemedScrollbars, Carousel, ButtonToggle } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import TradeCategories from 'Assets/Trading/Categories/trade-categories.jsx';
 import TradeCategoriesGIF from 'Assets/Trading/Categories/trade-categories-gif.jsx';
 import { getContractTypes } from '../../../../Helpers/contract-type';
+import ContractTypeGlossary from './contract-type-glossary.jsx';
+import classNames from 'classnames';
 
 const Info = ({ handleNavigationClick, handleSelect, initial_index, item, list }) => {
+    const [carousel_index, setCarouselIndex] = React.useState('');
+    const [selected_tab, setSelectedTab] = React.useState('description');
     const contract_types = getContractTypes(list, item).filter(i => i.value !== 'rise_fall_equal');
+    const onClickGlossary = () => setSelectedTab('glossary');
+
+    const has_toggle_buttons = carousel_index !== 'accumulator' && carousel_index !== 'vanilla';
+    const is_description_tab_selected = selected_tab === 'description';
+    const is_glossary_tab_selected = selected_tab === 'glossary';
```

Comment:
@henry-deriv let's introduce a constant for string values of tabs since we use 'glossary' and 'description' strings multiple times 🙂:
```
// declaration:
const TABS = {
    DESCRIPTION: 'description',
    GLOSSARY: 'glossary',
};
// usage:
const is_glossary_tab_selected = selected_tab === TABS.GLOSSARY; // and in all such places instead of strings ```